pkg/ebpf/tracee.go

```diff
@@ -59,6 +59,44 @@ type fileExecInfo struct {
 	Hash      string
 }
 
+type fileId struct {
```

Comment:
I would have made a `fileId` struct for the first 3, and envelope it with another type.
I see us in the future use the 3 file identifiers for other places.